nss-sapphire-elephants/BangazonSite
# Ticket three

# Description
Guest user cannot see the "Sell Something" link
Authenticated user can see the "Sell Something" link

When "Sell Something" is clicked, the Create Product form appears.
When the Create button is clicked, the new product is added to the DB. 
Create button redirects to display the details page for that new item.

## Type of change


# Testing Instructions for Change Made

```git fetch --all```

```git checkout ticket_three```

Open and note a guest will be missing the link in the nav bar but appears when a user logs in.
Note the form displays and a user is required to fill out all fields aside from "City"
Note that the product you created has been added

# Checklist:
